### ISSUE TITLE
Internal: rename example data directory

### ISSUE THREAD

**Issue by francisbarton** [ThomUK/SPCreporter#106]
The [R Packages book](https://r-pkgs.org/data.html#sec-data-extdata) suggests example data files be stored under `inst/extdata`.

**Comment by ThomUK** [comment]
Yes, agree.  I'm also conflicted about the use of the example data to run tests against.  I like the theory, but the practice becomes a bit unwieldy during development.  I'd welcome a discussion about the pros/cons of potentially separating the two:
1. Example data to show the package features to new users
2. A separate group of data files to check edge cases as part of testing

**Comment by francisbarton** [comment]
I think you're right that the data files for testing purposes (could be complex cases and with improbable data patterns) should be different to the example Excel files for users to get started with.

I don't know what the best practice is around data sets specifically for testing - internal package data, snapshots, or just functions/explicit data creation within the test suite itself? Whatever makes development go most smoothly. But yes go ahead and separate these, it was a nice idea in a way but it's not serving us in ongoing development.
